test/sourcemaps/index.ts

```diff
@@ -1,73 +1,120 @@
 import * as fs from "fs";
 import * as path from "path";
 import * as assert from "assert";
-import { svelte } from "../helpers";
+import { loadConfig, svelte } from "../helpers.ts";
```

Comment:
hmm. I'm not sure you rebased against `master` yet because I don't think the double quotes would be allowed here

This file has all single quotes in `master`:
https://github.com/sveltejs/svelte/blob/master/test/sourcemaps/index.ts

Reply:
did not rebase yet, i only merged the import file extension `.ts`

my plan for rebase is: squash all into two commits:
one to 'add feature preprocessor sourcemaps'
and the other to 'refactor tests'

Reply:
now revoked the merge, will handle changes in rebase

Reply:
rebase is here
https://github.com/milahu/svelte/tree/add-preprocessor-sourcemaps-rebased-to-ebbbc0d

Reply:
You don't need to make a new PR. You can just do `git push -f` to update this one